File: c++/iniciante/1036.cpp
#include <iostream>
#include <iomanip>
#include <cmath>

using namespace std;
 
int main() {
 
    double a,b,c,delta;
    
    cout<<fixed<<setprecision(1);
    cin>>a>>b>>c;
    delta = sqrt(b*b-4*a*c);
    
    if (b*b-4*a*c<0 || a==0) {
        cout<<"Impossivel calcular"<<endl;
    }
 
    else{
        cout<<fixed<<setprecision(5);
        cout<<"R1 = "<<((-b+delta)/(2*a))<<endl;
        cout<<"R2 = "<<((-b-delta)/(2*a))<<endl;
    }
    
    return 0;
}

File: python 3/iniciante/1036.py
#1036 - Fórmula de Bhaskara
from math import sqrt
a, b, c = map(float, input().split())
delta = (b*b)-(4*a*c)
if delta>=0:
    d = sqrt(delta)
    if(a==0):
        print('Impossivel calcular')
    else:
        raiz1 = (-b+d)/(2*a)
        raiz2 = (-b-d)/(2*a)
        print('R1 = {:.5f}'.format(raiz1))
        print('R2 = {:.5f}'.format(raiz2))
else:
    print('Impossivel calcular')
